gcp/table_gcp_kubernetes_node_pool.go

```diff
@@ -32,7 +32,7 @@ func tableGcpKubernetesNodePool(ctx context.Context) *plugin.Table {
 				Type:        proto.ColumnType_STRING,
 			},
 			{
-				Name:        "selfLink",
+				Name:        "self_link",
```

Comment:
This should be in a different PR since it's a bug fix

Reply:
will fix it in different PR.